content/blog/lifehacking/2017-12-01_freedom_and_productivity.md

```diff
@@ -0,0 +1,199 @@
+follow, and thus the practice of productivity routines will be aborted very
+soon after they get started. This is often a very strong sign for myself, when
+I consider doing something, add it to a todo list and months later it is still
+there; maybe it wasn't that useful or valuable, and I can erase it with a light
+heart.
+
+And when the initial, kickstarting motivation is right there, the *why* should
+also help set a limit to the endless quest of productivity. To know when to
+stop. For instance, when you managed to get more time for a dedicated hobby,
+and you're plain satifised, what would be the point of squeezing even more in
+your limited free time? Remember, productivity is a means to an end, not an end
+by itself. Without this in mind, you could be chasing an endless goal.
+
+People get easily fooled into thinking that more working hours imply more
+quality work. That's a societal feeling: in a capitalistic society, being
+productive means contributing to growth thus the good economic health of a
+entity greater than yourself (your country, your region, the world). And since
+people mix up being productive with working long hours, the confusion is easy
+to make. But I think productivity is the opposite of this: trying to work the
+least number of hours to advance work on the most important and valuable
+things. Begin maximally efficient with the minimum amount of resources.
+Identify those actions which will have the most positive impact, help the
+greatest number of people, be *valuable*. So it is another time to acknowledge
+the need for a why: why do you work for, in the first place? Is the work you're
+doing valuable, or are you doing busy work (like handling emails, answering
+very quickly on instant chat systems, etc.) to make it look like you're
+productive? Or more humbly, one might just be working to get the regular pay
+check, since work doesn't have to be a source of fulfillment, but instead can
+just be a means to an end too. Again, all motives are good here; but there
+should be one to make sense out of it.
+
+## To do what you want, to want what you do
+
+I think that's one goal of productivity: to be more conscious of all the
+choices we're making (what to work on) so that our work is more in adequation
+with our principles and real objectives. Thus it is about freedom, that is, the
+way to express what you truly desire to do and then to do it with purpose. Or
+in the words of French writer and journalist Françoise Giroud: happiness is
+doing what you want, and wanting what you do. I think other people call this
+autonomy or even integrity; I hope you get the idea.
+
+In particular, I don't want others to preconceive life choices for myself. Even
+if they have theoritized the topic and proven with objective metrics it worked
+for them. So here is my meta-productivity advice: to try different methods,
+and to make your own solutions out of them, to choose what works according to
```

Comment:
Remark: You could make this sentence bold characters. Just like the one above I said could be your title. They're important and may be the two phrases I'd want to read if I could only read two phrases of your article (which would be quite a shame!).

EDIT: Or highlight it as an out-of-text big side thing (but please not too far from the actual occurence, that's painful).